src/com/content-games/games.js

```diff
@@ -152,14 +156,28 @@ export default class ContentGames extends Component {
 				return <ContentItemBox node={r.node} user={props.user} path={props.path} noevent={props.noevent ? props.noevent : null} />;
 			});
 
+			while (Games.length % layout !== 0) {
+				Games.push(<ContentItemBox placeHolder={true} />);
+			}
+
 			if ( !props.nomore /*|| added >= 10*/ ){
 				LoadMore = <ContentMore onclick={this.fetchMore} />;
 			}
 
+			const gridClass = `-columns-${layout}`;
```

Comment:
Template strings (``) aren't supported in IE11 (http://caniuse.com/#feat=template-literals), and I don't believe we have `dangerousTaggedTemplateString` enabled in our Buble config (because like the name suggests, there's a danger to it).

Prefer a simple string append. i.e. `"-columns-"+layout`  or `'-columns-'+layout`.

Reply:
IE is why we can't have nice... but in this case non-formatted string was kind of fine too.

Reply:
Yeah this isn't so bad. `` is brand new. Now many people it's a feature (yet).